index.js

```diff
@@ -4,6 +4,7 @@ const got = require('got');
 const { BITBUCKET_USERNAME, BITBUCKET_PASSWORD, RENOVATE_BOT_USER } =
   process.env;
 const MANUAL_MERGE_MESSAGE = 'merge this manually';
+const AUTO_MERGE_MESSAGE = "🚦 **Automerge**: Enabled."
```

Comment:
```suggestion
const AUTO_MERGE_MESSAGE = "**Automerge**: Enabled."
```
I would remove the emoji, it could be an emoji string 

https://docs.renovatebot.com/self-hosted-configuration/#unicodeemoji